src/rantanplan/alternative_syllabification.py

```diff
@@ -1,3 +1,77 @@
+"""
+Exceptions for foreign words in Spanish that do not follow
+standard Spanish syllabification rules
+
+Exaxmple: "maître": "maî-tre"
+
+"""
+
+SYLLABIFICATOR_FOREIGN_WORDS_DICT = {
+    "aarónica": "aa-ró-ni-ca",
+    "aarónico": "aa-ró-ni-co",
+    "aaronita": "aa-ro-ni-ta",
+    "ampère": "am-pè-re",
```

Comment:
Is it not `am-père` in French pronunciation? I guess that list need to be thoroughly review at some point.

Reply:
I've done my homework :)
https://dirae.es/palabras/amp%C3%A8re